src/renderers/testing/ReactTestRendererFiberEntry.js

```diff
@@ -611,6 +605,21 @@ var ReactTestRendererFiber = {
         return TestRenderer.getPublicRootInstance(root);
       },
     };
+
+    Object.defineProperties(entry, {
+      root: {
+        get: function() {
+          if (root === null || root.current.child === null) {
+            throw new Error("Can't access .root on unmounted test renderer");
+          }
+          return wrapFiber(root.current.child);
+        },
+        enumerable: true,
+        configurable: true,
```

Comment:
It felt weird for a getter that might throw to be enumerable. I don't hold this opinion strongly though. 😄